client/pyproject.toml

```diff
@@ -0,0 +1,114 @@
+[project]
+# https://packaging.python.org/en/latest/specifications/declaring-project-metadata/
+name = "rubin-nublado-client"
+description = "Client for JupyterHub/JupyterLab in the Rubin Science Platform context"
+license = { file = "LICENSE" }
+readme = "README.md"
+keywords = ["rubin", "lsst"]
+# https://pypi.org/classifiers/
+classifiers = [
+    "Development Status :: 5 - Production/Stable",
+    "License :: OSI Approved :: MIT License",
+    "Programming Language :: Python",
+    "Programming Language :: Python :: 3",
+    "Programming Language :: Python :: 3.12",
+    "Intended Audience :: Developers",
+    "Natural Language :: English",
+    "Operating System :: POSIX",
+    "Typing :: Typed",
+]
+requires-python = ">=3.12"
+# Use requirements/main.in for runtime dependencies instead.
+dependencies = []
+dynamic = ["version"]
+
+[project.scripts]
+rsp_client = "rubin.nublado.client.cli:main"
```

Comment:
I don't this this `cli` module exists? So this'll need to be dropped in the first release.

Reply:
Oh, yeah, that was something I never finished.  Do you think it's a better idea to:

1) have a CLI class, and have a nublado client that you associate a username, token, and base URL with (or alternatively, token and base URL and ask Gafaelfawr for the name), and then have a CLI client, or
2) Assume that if you're using this class, you're happy with it being a Python object and not having an immediate CLI beyond the REPL.

I had started at 1, but never actually wrote the CLI tool, and after writing a bunch of hooks in Ghostwriter, find I don't miss it.

Reply:
Removed for now.  I'd be willing to listen to an argument for 1, but I'm not sure there's any reason for anyone, even me, to have a command-line tool to run code in a particular RSP as a particular user.  Noteburst if you love terminal windows, maybe?